### PR TITLE
Developer docs footer update.

### PR DESCRIPTION
**Pull request by rob-aitchison**

### CODE REVIEW

**Comment by rob-aitchison** [comment]
@000paradox000 , I tried to test this but it did not output what I wanted locally. @staceyoue let's discuss how you tested your tk-toolchain updates to ensure what you were seeing is accurate. Thanks!

**Review by 000paradox000 — APPROVED** [review]
Thanks LGTM

File: tk_toolchain/cmd_line_tools/tk_docs_generation/sphinx_data/_templates/layout.html [hunk 1]
```diff
@@ -66,10 +66,10 @@
 
     <b>{{project}}</b> {{release}}.<br>
     {% if project.startswith('tk-') %}
-        This documentation is part of the Shotgun Pipeline Toolkit.
+        This documentation is part of the ShotGrid Pipeline Toolkit.
     {% endif %}
     For more information, please visit
-    <a class=custom_post_menu href='https://developer.shotgridsoftware.com'>Shotgun Support</a>.
+    <a class=custom_post_menu href='https://developer.shotgridsoftware.com'>The ShotGrid developer portal.</a>.
```
Comment (000paradox000) [inline tk_toolchain/cmd_line_tools/tk_docs_generation/sphinx_data/_templates/layout.html:72]: ShotGrid developer documentation?

Comment (rob-aitchison) [reply tk_toolchain/cmd_line_tools/tk_docs_generation/sphinx_data/_templates/layout.html:72]: I prefer the term portal as it points to "lots" of documentation.

**Pull request merged**